src/crab/wto.cpp

```
@@ -142,4 +153,84 @@ wto_t::wto_t(const cfg_t& cfg) : _cfg(cfg) {
+
+std::vector<label_t> wto_t::collect_heads(const label_t& label) {
+    std::vector<label_t> heads;
+    for (auto h = head(label); h; h = head(*h)) {
+        heads.push_back(*h);
+    }
+    return heads;
+}
+
+// Compute the set of heads of the nested components containing a given label.
+// See section 3.1 of the paper for discussion, which uses the notation w(c).
+const wto_nesting_t& wto_t::nesting(const label_t& label) {
```

cppcheck

[style] The function 'nesting' is never used.

(unusedFunction)